### DISCUSSION TITLE
Unexpected vulnerability detected in package(nss)

### DISCUSSION THREAD

**Discussion by JwishPark** [aquasecurity/trivy#5401] [False Detection]
### IDs

CVE-2014-3566

### Description

Hello,
I scanned the vulnerability in my image(centos:7.9.2009) with Trivy.
And I found trivy detected CVE-2014-3566 in nss package.

However, when searching the [NVD official link](https://nvd.nist.gov/vuln/detail/CVE-2014-3566) and [docker hub](https://hub.docker.com/layers/library/centos/7.9.2009/images/sha256-dead07b4d8ed7e29e98de0f4504d87e8880d4347859d839686a31da35a3b532f?context=explore),
CVE-2014-3566 does not exist in nss package.

If follow the [PrimaryURL](https://avd.aquasec.com/nvd/cve-2021-43527) of the vulnerability check result,
it seems that trivy found the vulnerability based on the datasource from Vendor(Ubuntu).

I think this is wrong detection because the vulnerabilities that exist in the package may be different for each OS.
please check why trivy detected it.

### Reproduction Steps

```bash
1. docker pull centos:7.9.2009
2. trivy -d image --skip-update --offline-scan centos:7.9.2009
```

### Target

Container Image

### Scanner

Vulnerability

### Target OS

centos:7.9.2009

### Debug Output

```bash
trivy -d image --skip-update --offline-scan centos:7.9.2009
```
![image](https://github.com/aquasecurity/trivy/assets/103913389/fdf9131f-b30b-4c41-addf-86267a89e962)

### Version

```bash
$ trivy --version
Version: 0.41.0
Vulnerability DB:
  Version: 2
  UpdatedAt: 2023-10-12 00:26:18.957705896 +0000 UTC
  NextUpdate: 2023-10-12 06:26:18.957705396 +0000 UTC
  DownloadedAt: 0001-01-01 00:00:00 +0000 UTC
Java DB:
  Version: 1
  UpdatedAt: 2023-05-31 00:54:27.316128264 +0000 UTC
  NextUpdate: 2023-06-03 00:54:27.316127764 +0000 UTC
  DownloadedAt: 0001-01-01 00:00:00 +0000 UTC
```

### Checklist

- [X] Read [the documentation regarding wrong detection](https://aquasecurity.github.io/trivy/dev/community/contribute/discussion/#false-detection)
- [X] Ran Trivy with `-f json` that shows data sources and confirmed that the security advisory in data sources was correct

**Comment by DmitriyLewen** [comment]
Hello @JwishPark 
We use RedHat advisory database for CensOS - https://aquasecurity.github.io/trivy/v0.46/docs/scanner/vulnerability/#data-sources
RedHat marks `nss` as vulnerable package for RHEL7 - https://access.redhat.com/security/cve/cve-2014-3566
